src/main/java/org/mule/extension/ftp/internal/connection/FtpConnectionProvider.java

```diff
@@ -126,6 +126,15 @@ public String getWorkingDir() {
   @Summary("Whether to use passive mode. Set to \"false\" to switch to active mode")
   private boolean passive = true;
 
+  /**
+   *  Enable/disable verification that the remote host taking part of a data connection is the same as the host to which the control connection is attached.
+   */
+  @Parameter
+  @Optional(defaultValue = "true")
+  @Summary("Whether to use verify if remote host taking part of a data connection is the same as the host to which the control connection")
```

Comment:
"Whether to verify if the remote host taking part of a data connection is the same as the host to which the control connection is attached or not"

Reply:
Done!